.github/workflows/release-wheel-linux.yml

```diff
@@ -0,0 +1,259 @@
+          repository: ${{ inputs.repository  }}
+          script: ${{ inputs.pre-script }}
+      - name: Build clean
+        working-directory: ${{ inputs.repository }}
+        shell: bash -l {0}
+        run: |
+          set -x
+          source "${BUILD_ENV_FILE}"
+          ${CONDA_RUN} python setup.py clean
+      - name: Build the wheel (bdist_wheel)
+        working-directory: ${{ inputs.repository }}
+        shell: bash -l {0}
+        run: |
+          set -x
+          source "${BUILD_ENV_FILE}"
+
+          # BUILD_VERSION example: 2.4.0+cu121, we don't want the +cu121 part, so remove +cu121
+          BUILD_VERSION=${BUILD_VERSION%+*}
+          TRT_VERSION=10.0.1
+          mkdir release
+          mkdir release/tarball
+          mkdir release/wheel
+          if [[ "${{ inputs.cxx11-tarball-release }}" == "true" ]]; then
+              bazel build //:libtorchtrt --compilation_mode opt --config=default
+              cp bazel-bin/libtorchtrt.tar.gz \
+              release/tarball/libtorchtrt-${TORCHTRT_VERSION}-tensorrt${TENSORRT_VERSION}-cuda${CU_VERSION}-libtorch${PYTORCH_VERSION}-x86_64-linux.tar.gz
+          else
+              ${CONDA_RUN} python setup.py bdist_wheel --release
+              cp bazel-bin/libtorchtrt.tar.gz \
+              release/tarball/libtorchtrt-${BUILD_VERSION}-pre-cxx11-abi-tensorrt${TRT_VERSION}-cuda${CU_VERSION}-libtorch${BUILD_VERSION}-x86_64-linux.tar.gz
+
+              ${CONDA_RUN} python -m pip install auditwheel
+              ${CONDA_RUN} python -m auditwheel repair \
+              $(cat py/ci/soname_excludes.params) \
+                --plat manylinux_2_34_x86_64 \
+                dist/torch_tensorrt-*-linux_x86_64.whl
+              ${CONDA_RUN} python -m zipfile --list wheelhouse/torch_tensorrt-*_x86_64.whl
+              cp wheelhouse/torch_tensorrt-*_x86_64.whl release/wheel/
+          fi
+      - name: Run Post-Script
+        if: ${{ inputs.post-script != '' }}
+        uses: ./test-infra/.github/actions/run-script-with-cache
+        with:
+          repository: ${{ inputs.repository  }}
+          script: ${{ inputs.post-script }}
+      - name: Smoke Test
+        shell: bash -l {0}
+        env:
+          PACKAGE_NAME: ${{ inputs.package-name }}
+          SMOKE_TEST_SCRIPT: ${{ inputs.smoke-test-script }}
+        run: |
+          set -x
+          source "${BUILD_ENV_FILE}"
+          # TODO: add smoke test for the auditwheel tarball built
+
+      # NB: Only upload to GitHub after passing smoke tests
+      - name: Upload wheel to GitHub
+        if: ${{ inputs.cxx11-tarball-release != 'true' }}
+        continue-on-error: true
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ env.ARTIFACT_NAME }}
+          path: ${{ inputs.repository }}/release/wheel/
+      - name: Upload pre-cxx11 tarball to GitHub
+        if: ${{ inputs.cxx11-tarball-release != 'true' && env.PYTHON_VERSION == '3.10' && env.CU_VERSION == 'cu124' }}
```

Comment:
What is this check for?

Reply:
release-wheel-linux.yml is shared by release wheel and tarball:
1. release wheel uses python version: 3.8---3.12
2. release tarball is only for python 3.10
this check is for release pre-cxx11 tarball:
env.CU_VERSION == 'cu124' is not actually required, let me remove it.